structify/structify_test.go

```diff
@@ -0,0 +1,317 @@
+package structify
+
+import (
+	"github.com/icinga/icinga-go-library/types"
+	"github.com/stretchr/testify/require"
+	"reflect"
+	"testing"
+)
+
+type initerTest = struct {
```

Comment:
```suggestion
type initerTest struct {
```

There is no other occurrence of the `type T = struct` pattern in this code base. Please keep it consistent.